packages/components/row/src/row.ts

```diff
@@ -24,34 +25,36 @@ export const rowProps = buildProps({
 } as const)
 export type RowProps = ExtractPropTypes<typeof rowProps>
 
-export default defineComponent({
+const Row = defineComponent({
   name: 'ElRow',
   props: rowProps,
 
   setup(props, { slots }) {
+    const ns = useNamespace('row')
+
     const gutter = computed(() => props.gutter)
     provide('ElRow', {
       gutter,
     })
 
-    const style = computed(() => {
-      const ret = {
+    const style = computed<CSSProperties>(() => {
```

Comment:
I think the code is a little bit duplicated here since you have marked the return of this computed as `CSSProperties` which means the variable that you returned from the function body is inferred as `CSSProperties`, so that you don't need to explicitly declare that the `styles` is `CSSProperties`

Reply:
Fixed.